business_requirement/tests/test_br.py

```diff
@@ -113,19 +113,21 @@ def test_create_sequance(self):
             'name': '/',
             'description': 'test',
         }
-        self.br.create(br_vals)
+        res = self.br.create(br_vals)
+        self.assertTrue(res.name)
```

Comment:
@YogeshMahera-SerpentCS 

Ok let's try from the beggining.

1. Test need to be called with the name of function you want cover.
2. Them from my point of view the name is wrong should be `test_create`
3. Analyse the function. 
4. Found one if condition then means you need cover that path and the case that is False. 
```
@api.model
def create(self, vals):
# From here is test_create_name_equal_slash
    if vals.get('name', '/') == '/':
        vals['name'] = self.env['ir.sequence'].get('business.requirement')
        vals['name'] = self.env['ir.sequence'].next_by_code(
            'business.requirement')
############################################

# From here is test_create_project_id
    if vals.get('project_id'):
        project_id = self.env['project.project'].\
        project_id = self.env['project.project']. \
            browse(vals.get('project_id'))
############################################

# From here is test_create_project_id_message_follower_ids

        if project_id and project_id.message_follower_ids:
            vals['message_follower_ids'] =\
                project_id.message_follower_ids.ids
            msg_followers = []
            for project in project_id.message_follower_ids:
                msg_vals = {
                    'channel_id': project.channel_id.id,
                    'display_name': project.display_name,
                    'partner_id': project.partner_id.id,
                    'res_model': self._name,
                    'subtype_ids': project.subtype_ids.ids
                }
                msg_followers.append((0, 0, msg_vals))
############################################

# From here is test_create_project_id_message_follower_ids_msg_followers
            if msg_followers:
                vals['message_follower_ids'] = msg_followers
############################################

    return super(BusinessRequirement, self).create(vals)
```

So the first test for me would be.
```
def test_create_name_equal_slash(self, vals):
    name = self.env['ir.sequence'].get('business.requirement')
    name = self.env['ir.sequence'].next_by_code(
        'business.requirement')
    br_vals = {
        'name': '/',
        'description': 'test',
    }
    res = self.br.create(br_vals)
    self.assertEqual(name, res.name)
```

This is my idea about how should be but open to suggestions

cc @elicoidal @seb-elico @sudhir-serpentcs